processing/src/main/java/io/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class JavaScriptAggregatorFactory implements AggregatorFactory
+public class JavaScriptAggregatorFactory extends AggregatorFactory
```

Comment:
If we do want to do it that way, then that means that when we merge aggregatorFactories, we should do it in a particular order since the operation will not be commutative.